### ISSUE TITLE
Status color migration fails

### ISSUE THREAD

**Issue by elivz** [solspace/craft3-freeform#1561]
### What happened?

When updating to 5.6, there is a [migration](https://github.com/solspace/craft-freeform/blob/f748d1b13206482c9a396e55069e33aba03caa72/packages/plugin/src/migrations/m240813_161214_UpdateStatusColors.php#L27-L37) that resets several of the submission status colors. However, it fails because it tries to set them to `gray`, while `grey` is the acceptable spelling in the model enum.

### Errors and Stack Trace (if available)

```
*** applying m240813_161214_UpdateStatusColors
    > update in {{%freeform_statuses}} ...Exception: SQLSTATE[01000]: Warning: 1265 Data truncated for column 'color' at row 1
The SQL being executed was: UPDATE `craft_freeform_statuses` SET `color`='gray' WHERE `id`=1 (/var/www/html/vendor/yiisoft/yii2/db/Schema.php:676)
#0 /var/www/html/vendor/yiisoft/yii2/db/Command.php(1325): yii\db\Schema->convertException()
#1 /var/www/html/vendor/yiisoft/yii2/db/Command.php(1120): yii\db\Command->internalExecute()
#2 /var/www/html/vendor/craftcms/cms/src/db/Migration.php(270): yii\db\Command->execute()
#3 /var/www/html/vendor/solspace/craft-freeform/packages/plugin/src/migrations/m240813_161214_UpdateStatusColors.php(29): craft\db\Migration->update()
#4 /var/www/html/vendor/craftcms/cms/src/db/Migration.php(50): Solspace\Freeform\migrations\m240813_161214_UpdateStatusColors->safeUp()
#5 /var/www/html/vendor/yiisoft/yii2/console/controllers/BaseMigrateController.php(758): craft\db\Migration->up()
#6 /var/www/html/vendor/craftcms/cms/src/console/controllers/MigrateController.php(382): yii\console\controllers\BaseMigrateController->migrateUp()
```

### How can we reproduce this?

1. Install Freeform 5.5.x
2. Set any of the submission status's colors to `grey`, `light`, or `turquoise`
3. Update Freeform to 5.6.x and run pending migrations

### Freeform Edition

Pro

### Freeform Version

5.6.4

### Craft Version

5.4.7

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

5.5.7

**Comment by kjmartens** [comment]
Thanks for this, @elivz. We'll have this corrected for the next version of Freeform. 🙂

**Comment by kjmartens** [comment]
This should now be resolved in Freeform 5.6.5+ 🙂
